### PR TITLE
fix(vndr user repos) add support for user specified package locations (#316)

### PR DESCRIPTION
**Pull request by zlav**

### CODE REVIEW

File: buildtools/vndr/vndr_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package vndr_test
+
+import (
+	"testing"
+
+	"github.com/fossas/fossa-cli/buildtools"
+	"github.com/fossas/fossa-cli/buildtools/vndr"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/stretchr/testify/assert"
```
Comment (liftM) [inline buildtools/vndr/vndr_test.go]: [nit] Import grouping.

I think GolangCI might catch this for us when configured, but not by default. Might be worth looking into that, but that configuration should be a separate PR.

Comment (zlav) [reply buildtools/vndr/vndr_test.go]: This is actually a known [issue](https://github.com/golang/go/issues/20818) in goimports at the moment that is on their roadmap. I always thought I was safe running `goimport` `gofmt` `golint` in my shell before pushing. [Link](https://github.com/aristanetworks/goarista/tree/master/cmd/importsort) to an example that fixes this problem at least before commits.
